packages/twenty-emails/src/constants/DefaultWorkspaceLogo.ts

```diff
@@ -0,0 +1,2 @@
+export const DEFAULT_WORKSPACE_LOGO =
+  'https://twentyhq.github.io/placeholder-images/workspaces/twenty-logo.png';
```

Comment:
This is 32x32 and therefore will be pixelized in this context, could we replace it on the placeholder-images by a slightly higher res?